### PR TITLE
chore: release v0.10.0

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,36 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is release that fixes a HTTP server regression where the `backlog` was configured to `128`
```
Comment (niklasad1) [inline CHANGELOG.md:11]:
```suggestion
v0.10.0 is release that fixes a regression in HTTP server where the `backlog` was configured to `128`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,36 @@ The format is based on [Keep a Changelog].
...
+by default it is configured to `1024`.
```
Comment (niklasad1) [inline CHANGELOG.md:12]:
```suggestion
from this release on the `default backlog` is set `1024` by using `tokio::net::TcpListener`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,36 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is release that fixes a regression in HTTP server where the `backlog` was configured to `128`
+from this release on the `default backlog` is set `1024` by using `tokio::net::TcpListener`.
+In addition, this release introduces a couple new APIs for users to manually configure sockets based on their usage.
```
Comment (niklasad1) [inline CHANGELOG.md:13]:
```suggestion
In addition, this release introduces a couple new APIs for users to manually configure the HTTP server via `build_from_tcp` and `build_from_hyper`
```

Comment (niklasad1) [inline CHANGELOG.md:13]:
```suggestion
In addition, this release introduces a couple new APIs to manually configure the HTTP server via `build_from_tcp` and `build_from_hyper`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,36 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is release that fixes a regression in HTTP server where the `backlog` was configured to `128`
+from this release on the `default backlog` is set `1024` by using `tokio::net::TcpListener`.
+In addition, this release introduces a couple new APIs to manually configure the HTTP server via `build_from_tcp` and `build_from_hyper`
+
+
```
Comment (niklasad1) [inline CHANGELOG.md:15]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,35 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is release that fixes a regression in HTTP server where the `backlog` was configured to `128`
```
Comment (niklasad1) [inline CHANGELOG.md:11]:
```suggestion
v0.10.0 is release a that fixes a regression in HTTP server where the `backlog` was configured to `128`, introduces a couple of new APIs and a few minor bug fixes.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,35 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is release a that fixes a regression in HTTP server where the `backlog` was configured to `128`, introduces a couple of new APIs and a few minor bug fixes.
+from this release on the `default backlog` is set `1024` by using `tokio::net::TcpListener`.
```
Comment (niklasad1) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is a release that fixes a regression in HTTP server where the `backlog` was configured to `128`, introduces a couple of new APIs and a few minor bug fixes.
```
Comment (niklasad1) [inline CHANGELOG.md:11]:
```suggestion
v0.10.0 is a release that fixes a regression in the HTTP server where the `backlog` was configured to `128` (this is also configurable in this release) , introduces a couple of new APIs and a few minor bug fixes.
```

**Review by lexnv — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog].
...
+### [Changed]
+- [proc macros]: only generate unsub method if not provided (#702)
+- [examples]: update pubsub examples [#705](https://github.com/paritytech/jsonrpsee/pull/705)
+- core: remove `Error::Request` variant [#717](https://github.com/paritytech/jsonrpsee/pull/717)
+- Replace async-channel [#708](https://github.com/paritytech/jsonrpsee/pull/708)
+- chore(deps): bump actions/checkout from 2.4.0 to 3 [#710](https://github.com/paritytech/jsonrpsee/pull/710)
+- CI: cache cargo hack installation [#706](https://github.com/paritytech/jsonrpsee/pull/706)
+- CI: try nextest [#701](https://github.com/paritytech/jsonrpsee/pull/701)
+- chore(deps): update tokio-util requirement from 0.6 to 0.7 [#695](https://github.com/paritytech/jsonrpsee/pull/695)
+- Move CI script to new location [#694](https://github.com/paritytech/jsonrpsee/pull/694)
```
Comment (TarikGul) [inline CHANGELOG.md:22]:
```suggestion
- CI: Move CI script to new location [#694](https://github.com/paritytech/jsonrpsee/pull/694)
```

Just to keep in line with the above, just a suggestion though.

**Review by TarikGul — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog].
...
+### [Fixed]
+- fix(client): close subscription when server sent `SubscriptionClosed` notification [#721](https://github.com/paritytech/jsonrpsee/pull/721)
+- fix(http client): set reuseaddr and nodelay. [#687](https://github.com/paritytech/jsonrpsee/pull/687)
+- fix(rpc module): unsubscribe according ethereum pubsub spec [#693](https://github.com/paritytech/jsonrpsee/pull/693)
+- http server: fix regression set backlog to 1024 [#718](https://github.com/paritytech/jsonrpsee/pull/718)
+- README.MD: fix link to `ws server` [#703](https://github.com/paritytech/jsonrpsee/pull/703)
+
+### [Added]
+- feat(http server): add new builder APIs `build_from_tcp` and `build_from_hyper` [#719](https://github.com/paritytech/jsonrpsee/pull/719)
+- feat: add `SubscriptionSink::pipe_from_try_stream` to support streams that returns `Result` [#720](https://github.com/paritytech/jsonrpsee/pull/720)
```
Comment (niklasad1) [inline CHANGELOG.md:33]:
```suggestion
- feat(servers): add `SubscriptionSink::pipe_from_try_stream` to support streams that returns `Result` [#720](https://github.com/paritytech/jsonrpsee/pull/720)
```

**Review by jsdw — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.10.0] - 2022-04-04
+
+v0.10.0 is a release that fixes a regression in the HTTP server where the `backlog` was configured to `128` (this is also configurable now), introduces a couple of new APIs and a few minor bug fixes.
```
Comment (niklasad1) [inline CHANGELOG.md:11]:
```suggestion
v0.10.0 is a release that fixes a regression in the HTTP server where the backlog was hardcoded to 128 (this is now set to 1024 by default but also configurable), introduces a couple of new APIs and a few minor bug fixes.

If your usage expects a high rate of new HTTP connections you are encouraged to update or manually configure the socket based on the traffic characteristics 
```

**Pull request merged**